framework/auth/views.py

```diff
@@ -229,6 +254,126 @@ def confirm_email_get(token, auth=None, **kwargs):
     ))
 
 
+@collect_auth
+def confirm_user_get(auth=None, **kwargs):
+    """Called at login to see if there are emails to add or users to merge.
+    methods: GET
+    """
+    # user = User.load(kwargs['uid'])
+    user = auth.user
+    verified_emails = []
+    for token in user.email_verifications:
+        if user.confirm_token(token):
+            if user.email_verifications[token]['confirmed']:
+                try:
+                    user_merge = User.find_one(Q('emails', 'iexact', user.email_verifications[token]['email']))
```

Comment:
See [this comment](https://github.com/CenterForOpenScience/osf.io/pull/5000/files#r57469484)